pkg/packagekit/wix/service.go

```diff
@@ -241,3 +230,18 @@ func (s *Service) Xml(w io.Writer) error {
 	return nil
 
 }
+
+// cleanServiceName removes characters windows doesn't like in
+// services names, and converts everything to snake case.
+func cleanServiceName(in string) string {
+	r := strings.NewReplacer(
+		"-", "_",
+		" ", "_",
+		".", "_",
+	)
+
+	snakeName := r.Replace(strings.TrimSuffix(in, ".exe") + "_svc")
+	defaultName := snaker.SnakeToCamel(snakeName)
```

Comment:
you can `return snaker.SnakeToCamel` and avoid the extra variable assignment.

Reply:
Yeah... I probably didn't for some now removed debugging. Simplified, thanks!